frontend/public/locales/en/apply-adult-child.json

```
@@ -162,24 +156,31 @@
-          "eligible-list": {
-            "employer": "an employer",
-            "pension": "pension (see exception)",
-            "professional": "professional or student organization."
-          },
-          "not-eligible": "As such, the child is not eligible for Canadian Dental Care Plan.",
-          "not-eligible-purchased": "If a parent or legal guardian purchased the current dental insurance policy privately, a child is not eligible for the Canadian Dental Care Plan while that coverage is in effect.",
-          "excepton": "Exception: A child may still be eligible if a parent or legal guardian are retired and:",
-          "exception-list": {
-            "opted-out": "Opted out of pension benefits before December 11, 2023, and",
-            "opt-back": "Can't opt back in under the pension rules"
+          "title": "What does it mean to have access to private dental insurance or coverage?",
+          "eligible": "It means the child has access to any type of dental insurance or coverage through:",
+          "list": {
+            "employer": "a parent or legal guardian's employment benefits, including health and wellness accounts",
+            "organization": "a professional or student organization",
+            "organization-note": "Note: If the child is eligible for dental coverage through a parent or legal guardian's employment benefits, or through a professional or student organization, they're not eligible for the CDCP. This is true even if:",
+            "organization-list": {
+              "decide": "you decide not to take it",
+              "premium": "you have to pay a premium for it",
+              "use": "you don't use it"
+            },
+            "pension": "a parent or legal guardian's pension benefits",
+            "pension-plans": "this includes federal, provincial and territorial government employer pension plans",
+            "pension-exemption": "Exception: The child may still be eligible if a parent or legal guardian is retired and:",
+            "pension-list": {
+              "opt-out": "opted out of pension benefits before December 11, 2023, and",
+              "opt-in": "can't opt back in under the pension rules"
+            },
+            "company": "coverage bought by a parent or legal guardian through a group plan from an insurance or benefits company.",
+            "cannot-opt": "if a parent or legal guardian purchased the child's current dental insurance policy privately (and not as part of any of the coverage described above), the child is not eligible for the CDCP while that coverage is in effect."
           }
         }
       },
       "error-message": {
-        "dental-insurance-required": "Select whether the child has access to dental insurance"
+        "dental-insurance-required": "Select whether you have access to dental insurance"
```

Same comment as above and for French as well.

```suggestion
        "dental-insurance-required": "Select whether the child has access to dental insurance"
```